src/build.d

```diff
@@ -357,8 +357,7 @@ alias dmdExe = memoize!(function(string targetSuffix, string[] extraFlags...)
             env["HOST_DMD_RUN"],
             "-of" ~ target,
             "-vtls",
-            "-J"~env["G"],
```

Comment:
We need `"-J" ~ env["G"]` to be able to string-import the `VERSION` file, or am I missing something?

Reply:
Its included in `flags["DFLAGS"`, see https://github.com/dlang/dmd/blob/cc78c274384608098a2a8ca25984903cb69078ef/src/build.d#L879